fido/fido.py

```diff
@@ -255,7 +254,6 @@ def fetch(
     body='',
     timeout=DEFAULT_TIMEOUT,
     connect_timeout=DEFAULT_CONNECT_TIMEOUT,
-    content_type=DEFAULT_CONTENT_TYPE,
```

Comment:
This is a breaking change, we should do an appropriate version bump.

Another way would be to leave the parameter and only set it if a body is provided. This would change the behavior (fix the bug) but be backwards compatible (not change the interface).